components/Stepper/Stepper.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+import { colors, baseFontSize, spacing } from '../shared/theme';
+import {
+  handlerValuePrepare,
+  handlerStepPrepare,
+  percentToDegrees,
+} from './helper';
+import {
+  Wrapper,
+  RadialProgressBar,
+  RadialProgressOverlay,
+  TextWrapper,
+  CurrentText,
+  NextStep,
+} from './style';
+
+RadialProgressOverlay.displayName = 'RadialProgressOverlay';
+NextStep.displayName = 'NextStep';
+
+const Stepper = ({
+  total,
+  index,
+  currentStepText,
+  nextStepText,
+  isPtBR,
+  ...rest
+}) => {
+  const _totalPrepared = handlerValuePrepare(total);
+  const _indexPrepared = handlerStepPrepare(index, total);
+  const _isLastStep = _totalPrepared === _indexPrepared;
+  const _progressPercent = Math.round((_indexPrepared / _totalPrepared) * 100);
+
+  const _nextText = isPtBR ? 'próximo: ' : 'next: ';
+  const _paginationText = isPtBR ? 'de' : 'of';
```

Comment:
what do you think to set this vars with a props instead of a conditional?

```suggestion
  const _nextText = settings.nextText || 'next';
  const _paginationText = settings.paginationText || 'of';
```

It's create a settings dynamic over "manual" instead.

Reply:
good! Made.